pkg/arvo/sys/vane/dill.hoon

```diff
@@ -149,13 +157,43 @@
         ^-  {(list move) axle}
         [(flop moz) all(dug (~(put by dug.all) hen +<+))]
       ::
+      ++  auto                                          ::  stage automation
+        ^+  .
+        ?.  dog.all  .
+        =.  dog.all  |
+        (pass /auto/one [%b %wait +(now)])
+      ::
+      ++  auto-wake                                     ::  resume automation
+        |=  [=wire error=(unit tang)]
+        ?+  wire
+          ?~  error
+            ~|(behn-bad-wake+wire !!)
+          (crud %wake u.error)
+        ::
+            [%auto %one ~]
+          ?~  error
+            ~&  %behn-goad
+            (pass / [%g %goad force=| ~])
+          ::  %goad crashed, wait again, then force
+          ::
+          (pass /auto/two [%b %wait +(now)])
```

Comment:
Would be nice to print u.error here.  It did get to behn-goad-again on my ship.

Reply:
That'll be a crash in ford due to invalid internal state (ie, `+got:by` on an anchor or sub-build). I figured it'd be confusing to print the crash since we're continuing, but it's a one-time thing so it's probably not a big deal. I'll add it.

Reply:
That makes sense.  Ideally you'd recognize that specific error and not print it, but it doesn't matter much.